content/applications/sales/crm/track_leads/marketing_attribution.rst

```diff
@@ -0,0 +1,157 @@
+
+:guilabel:`Filters`, located in the left column of the search options, can be used to keep only
+the results that fit the filter. For example, selecting the :guilabel:`Won` filter only
+shows leads that have been won in the attribution report.
+
+:guilabel:`Group By`, found in the middle column, is used to organize the results into groups, and
+can be used with or without filters.
+
+.. image:: marketing_attribution/search-results-multiple-options.png
+   :align: center
+   :alt: Select any number of filters and groups in the search options.
+
+.. tip::
+    Setting multiple :guilabel:`Group By` options creates nested groups, according to which option
+    is selected first. For example, selecting :guilabel:`Medium`, followed by :guilabel:`Source`,
+    and then :guilabel:`Campaign`, in :guilabel:`Group By` column, sorts all results *first* by
+    medium, *then* by the specific sources in each medium, followed by the campaigns in each source.
+
+    This can be verified by looking at the direction, and order, of the selections in the group tile
+    that appears in the :guilabel:`Search...` bar.
+
+    .. image:: marketing_attribution/group-by.png
+       :align: center
+       :alt: The text in the tile is `Country > City`, showing that city is a subgroup of country.
+
+.. example::
+    For a useful first report:
+
+    #. From the :guilabel:`Filters` column, select the :guilabel:`Active` filter to view only leads
+    that are still marked as active.
+    #. From the :guilabel:`Group By` column, select (in this specific order) :guilabel:`Source`,
+    followed by the :guilabel:`City` or :guilabel:`Country`, depending on which grouping is more
+    relevant.
+
+    .. image:: marketing_attribution/campaign-and-country-groups.png
+       :align: center
+       :alt: Each lead is now sorted by source, followed by city or country.
+
+    This report contains all active leads, grouped first by the source of the lead, then the
+    city or country each lead is from. This is useful to see the density of active opportunities
+    sorted by location.
+
+    With this data, marketing campaigns, such as conferences or billboards can be targeted to the
+    locations generating the largest amount of potential revenue. Similarly, more attention can be
+    put toward increasing outreach in locations where existing marketing campaigns are less
+    effective.
+
+Export reports
+==============
+
+To set the measures of the report, begin by navigating to the :icon:`oi-view-pivot`
+:guilabel:`(pivot view)`. Click the :guilabel:`Measures` button to view the available measures of
+the report. Select the desired measures from the drop-down menu (multiple measures can be selected),
+and verify the measures, filters, and groups are all displayed correctly in the pivot table. This
+ensures the data is ready for export.
+
+To quickly export the list as a .xlsx file, navigate to the :icon:`oi-view-list` :guilabel:`(list
```

Comment:
```suggestion
To quickly export the data in a list, as a .xlsx file, navigate to the :icon:`oi-view-list` :guilabel:`(list
```

Reply:
kept going back-and-forth between "an .xlsx file" and "a .xlsx file" -- it all depends on if the reader reads it as "dot-xlsx" or just "xlsx" -- totally your call. Just figured it was worth mentioning...